src/test/java/pages/LoginPopup.java

```diff
@@ -0,0 +1,28 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPopup {
+    private static final By LOGIN_LINK = By.className("enter");
+    private static final By LOGIN = By.name("login");
+    private static final By PASSWORD = By.name("password");
+    private static final By LOGIN_BUTTON = By.xpath(".//input[contains(@class, 'button auth__enter')]");
+    private static final By LOGGED_IN_RESULT = By.xpath(".//a[contains(@class, 'logedin')]");
+    private final WebDriver driver;
+
+    public LoginPopup() {
+        this.driver = WebDriverSingleton.getInstance();
+    }
+
+    public void login(String username, String password) {
```

Comment:
According to page object pattern, most of methods return current page or new page. It means, that if after login you are on the same page, then method must return current page object, otherwise another. In this case after login you are on another page, please return it (don't forget to update tests after that).